### ISSUE TITLE
[BUG] Insecure Browser

### ISSUE THREAD

**Issue by bannguycsventures** [liberodark/ODrive#163]
**Describe the bug**
Google shows an unsuported browser prompt after attempting to setup an account.

**To Reproduce**
1. Start the app
2. Setup an account
3. When the Google login screen shows, input a valid email
The login page would then show an "insecure browser" message

**Expected behavior**
User can login normally.

**Screenshots**
[https://live.staticflickr.com/65535/50236617351_99a2001095_z.jpg](insecure browser)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 LTS

**Comment by cheesycod** [comment]
This browser or app may not be secure. Learn more

Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in.

I can confirm this bug

**Comment by marques576** [comment]
I also have the same problem

**Comment by rodrigopequeno** [comment]
I also have the same problem

**Comment by frenchiveruti** [comment]
Me too, I'm having the same issue.  
Windows 10 Pro:
![imagen](https://user-images.githubusercontent.com/1095997/93938991-7d2ab680-fd00-11ea-851b-9f34512da452.png)
Running latest version 0.2.1 of ODrive.  
Same error as the OP.

**Comment by kilynet** [comment]
same problem

**Comment by TadeuBotelho** [comment]
Encontrei o mesmo problema aqui no ubuntu 20.04.
Vamos analisar o source para ajudar na resolução desse problema.

**Comment by TaoHuaLiu** [comment]
Same bug with ODrive 0.2.2 installed as a flatpak on Fedora 33.

**Comment by James-Livesey** [comment]
Also have the same issue. Anyone gonna fix this? (@liberodark )

Debian 10

**Comment by faturmaster** [comment]
Same problem here using Zorin OS and latest ODrive installation

**Comment by WilliamFreau** [comment]
Same on Ubuntu 20.10 with Snap package
![image](https://user-images.githubusercontent.com/38540996/100417972-394a8980-3082-11eb-9745-35fc7c1e8fd8.png)

**Comment by Epizefiri** [comment]
Same on ubuntu 20.04 with Snap

**Comment by vdjkelly** [comment]
Same bug with ODrive Debian 10
![image](https://user-images.githubusercontent.com/5873347/101688040-c9ed7600-3a41-11eb-9600-60e9479e2163.png)

**Comment by xanda-escuyer** [comment]
We also can confirm this on Windows 7 Ultimate x64
![oDrive](https://user-images.githubusercontent.com/15840995/101950803-c696ed80-3bed-11eb-8334-cf61fde27072.png)

**Comment by Daidalos117** [comment]
I was able to get it working by cloning distro and running locally (run `npm run release-linux` ). I was actually trying to get this fixed, but it seems that adding account in localy bundled app just works without problem. It's shame tho, because I cannot prove if my solution works. Anyone willing to collaborate?

**Comment by xanda-escuyer** [comment]
Happy to help if I can: I'm not a developer and Windows only but if you have a method or other procedure to test then by all means.

**Comment by brockers** [comment]
Same problem here on Fedora 33 build directly from npm.

**Comment by Yifei0727** [comment]
maybe help

> Information for app developers
>
> If you implemented "Sign in with Google" with the Chromium Embedded Framework, you’ll need to migrate to a more secure alternative:
>
> Switch to use [browser-based OAuth](https://developers.google.com/identity/protocols/oauth2/native-app).
Migrate your desktop app to a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/).

**Comment by LucasDondo** [comment]
Same here!
elementary OS 5.1.7 Hera, built on Ubuntu 18.04.4 LTS.
I tried with ODrive installed as DEB (0.2.1), AppImage (0.2.1) and Snap (`odrive` and `odrive-unofficial`). None worked.

**Comment by j2l** [comment]
same problem
uninstall for now

**Comment by jesbenmx** [comment]
Same problem and nothing solution 5 moth after, and the problem its google, because same problem in other apps in windows (super productivity), but the problem right now its nothing fix

**Comment by digibucc** [comment]
same, arch linux

**Comment by j2l** [comment]
Well, as it's written on the ReadMe:
Recommend to use VGrive as alternative of this app.

**Comment by Bradford1040** [comment]
MXLinux this is my problem as well, it obviously started on 3/20/21 as that was the last sync time. I tried uninstalling and reinstalling, clearing browser cache nothing seems to fix this issue,

**Comment by James-Livesey** [comment]
I'd suggest closing this issue since there doesn't seem to be much progress being made; only just other people complaining that it won't work (thanks to restrictions imposed on Google which are hard to get around on our side without a rewrite of the auth system).

It seems that the owner of this repo is pushing for us to use an alternative since this repo isn't really in active development any more, and I doubt that there are going to be any fixes issued in the near future.

---

(I'm not a maintainer of this repo btw, before anyone asks!)

**Comment by Bradford1040** [comment]
That stinks, but understandable, it was a really good app, been using for a really long time. Sorry to see it go.

**Comment by James-Livesey** [comment]
@Bradford1040 Yeah, it's a shame! Haven't tested VGrive, but it seems to be the way to go. According to the readme of this repo:

> Recommend to use VGrive as alternative of this app :
> https://github.com/bcedu/VGrive/

VGrive looks to be just as good as this, though personally I do think it has a bit of a strange name!

**Comment by liberodark** [comment]
Probably fixed by : https://github.com/liberodark/ODrive/pull/165

**Comment by josefranw** [comment]
I still can't log in.

**Comment by mahbd** [comment]
Also not working on ubuntu 21.04

![Screenshot from 2021-06-01 13-13-38](https://user-images.githubusercontent.com/53579616/120282489-f9d45280-c2db-11eb-8e53-16723cc512a7.png)
